cli/azd/pkg/pipeline/pipeline_manager_test.go

```diff
@@ -339,6 +394,13 @@ func Test_PipelineManager_Initialize(t *testing.T) {
 		assert.NotNil(t, manager)
 		assert.NoError(t, err)
 
+		manager.infra = &project.Infra{
+			Options: provisioning.Options{
+				Provider: provisioning.Bicep,
+			},
+			IsCompose: false}
+		manager.configOptions = &configurePipelineOptions{}
```

Comment:
nit: Can we omit setting these empty options and have the manager handle this internally setting to empty options?